doc/sphinx/Makefile

```diff
@@ -75,5 +82,10 @@ clean-module-docs: FORCE
 	rm -rf source/modules/packages
 	rm -rf source/modules/standard
 
+clean-primers: FORCE
+	@echo
+	@echo "Removing primers generated into source/primers"
+	git clean -fX source/primers
```

Comment:
I don't necessarily disagree, but in what way exactly? I'd argue users shouldn't expect anything inside `source/` or `build/` to be safe against a `make clean`.

Furthermore, if we were OK with this approach, this could replace a lot of the other clean targets as well.